student.rb

```diff
@@ -1,19 +1,18 @@
-require_relative 'person'
+require_relative './person'
 
 class Student < Person
-  attr_reader :classroom
+  attr_reader :classrom
 
-  def initialize(classroom)
-    super(name, age, parent_permission)
-    @classroom = classroom
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] I had the same errors as the previous reviewer when trying to create a student. I fixed it by changing the super method to this. You need to specify the `parent_permission` like this since you are using the colon notation

  ```
  super(age, name, parent_permission: parent_permission)
  ```
  ![ddow](https://user-images.githubusercontent.com/36542986/193426457-f454606e-8362-45d4-8ad1-c53ab0cfadf4.jpg)

Reply:
@TedLivist thank you for the guidance